testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveCoercionOnUnpartitionedTable.java

```diff
@@ -101,6 +101,8 @@ protected Map<ColumnContext, String> expectedExceptionsWithTrinoContext()
     {
         // TODO: These expected failures should be fixed.
         return ImmutableMap.<ColumnContext, String>builder()
+                // Expected failures from BaseTestHiveCoercion
+                .putAll(super.expectedExceptionsWithTrinoContext())
```

Comment:
this is meaningless since super is empty right now.
squash with commit that changes super

Reply:
this is now squashed, but remains meaningless, as super remains empty.

maybe make super abstract instead?